### PR TITLE
Support for grouping fields in admin

### PR DESCRIPTION
**Pull request by remik**

### CODE REVIEW

**Comment by remik** [comment]
Please review.
It might need a bit better docs.

Hi from PyCon US 2016 :)

File: pages/placeholders.py [hunk 1]
```diff
@@ -62,6 +62,11 @@ def parse_placeholder(parser, token):
                 params['widget'] = remaining[1]
             if bit == 'on':
                 params['page'] = remaining[1]
+            if bit == 'in':
+                quotes = ('"', "'")
+                if remaining[1].startswith(quotes) and remaining[1].endswith(quotes):
+                    remaining[1] = remaining[1][1:-1]
```
Comment (batiste) [inline pages/placeholders.py]: You should use unescape_string_literal here

https://docs.djangoproject.com/en/1.9/_modules/django/utils/text/

**Comment by batiste** [comment]
Just a little comment otherwise this seems good to go! Great feature implemented rather simply.

**Comment by batiste** [comment]
Also what about being a little more explicit and use "group" as a keyword instead of "in"?

**Comment by remik** [comment]
I was thinking about "group by" but then it complicates parser code to get more tokens.
For me "group" looks weird but I can change if you want.

File: pages/placeholders.py (hunk 1 above)
Comment (remik) [inline pages/placeholders.py]: Changed. Thanks!

**Comment by batiste** [comment]
Django template don't seem to use underscore: e.g. truncatewords, filesizeformat, intcomma so I would not go for group_by. What do you think of section? That seems to describe things very well?

**Comment by remik** [comment]
Yep, "section" is good. I will use it and rebase commits so we will have one history only:)

**Comment by remik** [comment]
I know I can merge it but I will give you time for last check:)

**Pull request merged**